### ISSUE TITLE
This template cannot even start

### ISSUE THREAD

**Issue by hipertracker** [nuxt-community/typescript-template#22]
```
$ vue init nuxt-community/typescript-template myapp
$ cd myap
$ yarn

yarn install v1.3.2
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > nuxt-class-component@1.2.0" has unmet peer dependency "vue@^2.5.13".
warning " > nuxt-class-component@1.2.0" has unmet peer dependency "vuex@^3.0.1".
warning "nuxt-class-component > vuex-class@0.3.0" has unmet peer dependency "vue@^2.5.0".
warning "nuxt-class-component > vuex-class@0.3.0" has unmet peer dependency "vuex@^3.0.0".
warning " > vuex-class@0.2.0" has unmet peer dependency "vue@^2.0.0".
warning " > vuex-class@0.2.0" has unmet peer dependency "vuex@^2.0.0".
[4/4] 📃  Building fresh packages...
success Saved lockfile.
✨  Done in 14.31s.


$ yarn dev

yarn dev
yarn run v1.3.2
$ nuxt

 WARN  ~modules/typescript.ts

Supporting callbacks is depricated and will be removed in next releases. Consider using async/await.


 ERROR

  TypeError: Cannot read property 'loaders' of undefined

  - typescript.ts:12 Builder.extendBuild.config
    /Users/hipertracker/myapp/modules/typescript.ts:12:20

  - utils.js:112 Builder.<anonymous>
    [myapp]/[nuxt]/lib/common/utils.js:112:17

  - client.config.js:208 Builder.webpackClientConfig
    [myapp]/[nuxt]/lib/builder/webpack/client.config.js:208:54

  - builder.js:485 Builder.webpackBuild
    [myapp]/[nuxt]/lib/builder/builder.js:485:46

  - builder.js:168 Builder.build
    [myapp]/[nuxt]/lib/builder/builder.js:168:16
```

**Comment by wss610** [comment]
Change the "rule.query.loaders.ts = " to "rule.options.loaders.ts = "  in "\modules\typescript.ts"

**Comment by qm3ster** [comment]
fixed in #27
